README.md

```diff
@@ -1,66 +1,93 @@
+# ArcGIS Runtime SDK for .NET - Toolkit
 
-There are two ways to add Toolkit to your project:
+The ArcGIS Runtime SDK for .NET Toolkit contains controls and components to accelerate your development with [ArcGIS Runtime SDK for .NET](https://developers.arcgis.com/net/).
+
+[![Link: Toolkit Guide](https://img.shields.io/badge/%F0%9F%93%84-Toolkit%20Guide-633b9b?style=flat-square)](https://esri.github.io/arcgis-toolkit-dotnet/controls.html)
+[![Link: Toolkit API Reference (landing page)](https://img.shields.io/badge/Toolkit%20API%20Reference:-fff?style=flat-square)](https://esri.github.io/arcgis-toolkit-dotnet/api/index.html)
+[![Link: Toolkit API Reference - Android](https://img.shields.io/badge/Xamarin.Android-3ddc84?style=flat-square&labelColor=gray&logo=android)](https://esri.github.io/arcgis-toolkit-dotnet/api/android/index.html)
+[![Link: Toolkit API Reference - iOS](https://img.shields.io/badge/Xamarin.iOS-black?style=flat-square&labelColor=gray&logo=ios)](https://esri.github.io/arcgis-toolkit-dotnet/api/ios/index.html)
+[![Link: Toolkit API Reference - Forms](https://img.shields.io/badge/Xamarin.Forms-3498db?style=flat-square&labelColor=gray&logo=Xamarin)](https://esri.github.io/arcgis-toolkit-dotnet/api/netstd/index.html)
+[![Link: Toolkit API Reference - WPF](https://img.shields.io/badge/WPF-0078d6?style=flat-square&labelColor=gray&logo=windowsxp)](https://esri.github.io/arcgis-toolkit-dotnet/api/netfx/index.html)
+[![Link: Toolkit API Reference - UWP](https://img.shields.io/badge/UWP-0078d6?style=flat-square&labelColor=gray&logo=windows)](https://esri.github.io/arcgis-toolkit-dotnet/api/uwp/index.html)
+
+<table aria-hidden="true">
+<tr>
+<td>
+<img width="150" title="Basemap Gallery" src="https://user-images.githubusercontent.com/29742178/124198151-f2dc6380-da84-11eb-8e78-4e705d14c33d.png" />
+</td>
+<td>
+<img src="https://user-images.githubusercontent.com/29742178/121975740-34f07000-cd37-11eb-9162-462925cb3fe7.png" width="150" title="Overview Map" />
+</td>
+<td>
+<img src="https://user-images.githubusercontent.com/1378165/73389924-011fc200-428a-11ea-91bf-4ea1c2bf6683.png" width="105" title="Legend" />
+</td>
+<td>
+<img title="Search View" width="150" src="https://user-images.githubusercontent.com/29742178/142301018-4bbeb0f2-3021-49a7-b5ec-f642c5700bd0.png" />
+</td>
+</tr>
+</table>
 
-1. **Install from Nuget** - the fastest way to get toolkit into your app
-    - [Esri.ArcGISRuntime.Toolkit](https://www.nuget.org/packages/Esri.ArcGISRuntime.Toolkit)
-	- [Esri.ArcGISRuntime.Toolkit.Xamarin.Forms](https://www.nuget.org/packages/Esri.ArcGISRuntime.Toolkit.Xamarin.Forms)
-	- [Esri.ArcGISRuntime.ARToolkit](https://www.nuget.org/packages/Esri.ArcGISRuntime.ARToolkit)
-	- [Esri.ArcGISRuntime.ARToolkit.Forms](https://www.nuget.org/packages/Esri.ArcGISRuntime.ARToolkit.Forms)
-2. **[Build from source](https://esri.github.io/arcgis-toolkit-dotnet/buildingtoolkit.html)** - do this if you want to customize toolkit
-   
 ## Features
 
```

Comment:
How about moving the feature availability table to a separate section in the readme (or even a different doc) and instead include some brief doc about the features, including screenshots?